MdeModulePkg/Universal/SecurityStubDxe/Defer3rdPartyImageLoad.c

```diff
@@ -25,7 +25,7 @@ typedef struct {
 } DEFERRED_3RD_PARTY_IMAGE_TABLE;
 
 BOOLEAN                          mImageLoadedAfterEndOfDxe   = FALSE;
-BOOLEAN                          mEndOfDxe                   = TRUE;
+BOOLEAN                          mEndOfDxe                   = FALSE;

```

Comment:
We should not modify this code. There must be a different way/solution to fix the OptionROM issue. mEndOfDxe is probably being set to TRUE by an event. Search for this event and try to understand when it is set

Reply:
Ok actually I see it was modified before https://github.com/Dasharo/edk2/commit/976412e7dd778ce281c5a138cb00dbe54a293e67
So it is a correct solution after all.